politeiawww/invoices.go

```diff
@@ -384,6 +385,128 @@ func (p *politeiawww) processInvoiceDetails(invDetails cms.InvoiceDetails,
 	return &reply, nil
 }
 
+// processUserInvoices fetches all invoices that are currently stored in the
+// cmsdb for the logged in user.
+func (p *politeiawww) processUserInvoices(user *user.User) (*cms.UserInvoicesReply, error) {
+	log.Tracef("processUserInvoices")
+
+	dbInvs, err := p.cmsDB.InvoicesByUserID(user.ID.String())
+	if err != nil {
+		return nil, err
+	}
+
+	invRecs := make([]cms.InvoiceRecord, 0, len(dbInvs))
+	for _, inv := range dbInvs {
+		r, err := p.cache.Record(inv.Token)
+		if err != nil {
+			return nil, err
+		}
+
+		invRec, err := convertDatabaseInvoiceToInvoiceRecord(inv)
+		if err != nil {
+			return nil, err
+		}
+		invRec.Username = p.getUsernameById(invRec.UserID)
+
+		// Get raw record information from d cache
+		pr := convertPropFromCache(*r)
+
+		invRec.Files = pr.Files
+		invRec.CensorshipRecord = pr.CensorshipRecord
+		invRec.Signature = pr.Signature
+		invRecs = append(invRecs, *invRec)
+	}
+
+	// Setup reply
+	reply := cms.UserInvoicesReply{
+		Invoices: invRecs,
+	}
+	return &reply, nil
+}
+
+// processAdminInvoices fetches all invoices that are currently stored in the
+// cmsdb for an administrator, based on request fields (month/year and/or status).
+func (p *politeiawww) processAdminInvoices(ai cms.AdminInvoices,
```

Comment:
function declarations should only be one line.  They are the exception to the 80 column limit.  The reason is because it messes up `godoc` for exported functions and since we do it for exported functions, we do it for all functions in order to be consistent.